clean/new_orleans_da_cprr_2021_brady.py

```diff
@@ -0,0 +1,347 @@
+        .str.replace('rule2:moral conduct ; paragraph 01 adherence to law',
+                     'rule 2: moral conduct;paragraph 01 adherence to law', regex=False)\
+        .str.replace('rule 2: oral conduci ; paragraph 01 adherence to law',
+                     'rule 2: moral conduci;paragraph 01 adherence to law', regex=False)\
+        .str.replace('rule 2: ralconduci ; paragraph 01 adherence to law',
+                     'rule 2: moral conduct;paragraph 01 adherence to law', regex=False)\
+        .str.replace('rule 2:moral conduct ; paragraph 01 adherence to law',
+                     'rule 2: moral conduct;paragraph 01 adherence to law', regex=False)\
+        .str.replace('rule 2:moral conduct ; paragraph 03 honesty and truthfulness',
+                     'rule 2: moral conduct;paragraph 03 honesty and truthfulness', regex=False)\
+        .str.replace('rule 2:moral conduct ; paragraph adherence to law',
+                     'rule 2: moral conduct;paragraph 01 adherence to law', regex=False)\
+        .str.replace('rule: 2: moral conduct ; paragraph 01 adherence to law',
+                     'rule 2: moral conduct;paragraph 01 adherence to law', regex=False)\
+        .str.replace('rule 2: oral conduct ; paragraph 01 adherence to law',
+                     'rule 2: moral conduct;paragraph 01 adherence to law', regex=False)\
+        .str.replace('rule 2: oralconduct ; paragraph 01 adherence to law',
+                     'rule 2: moral conduct;paragraph 01 adherence to law', regex=False)\
+        .str.replace('rule 2: moral conduct ; paragraph 01 adherence to law',
+                     'rule 2: moral conduct;paragraph 01 adherence to law', regex=False)\
+        .str.replace('rule 2: moral conduct ; paragraph 03 honesty and truthfulness',
+                     'rule 2: moral conduct;paragraph 03 honesty and truthfulness', regex=False)\
+        .str.replace('rule 2: moral conduct ; paragraph 01 adherenceto law',
+                     'rule 2: moral conduct;paragraph 01 adherence to law', regex=False)\
+        .str.replace('rule 2: moral conduci;paragraph 01 adherence to law',
+                     'rule 2: moral conduct;paragraph 01 adherence to law', regex=False)\
+        .str.replace('rule 2 moral conduct ; paragraph 0 honesty and truthfulness',
+                     'rule 2: moral conduct;paragraph 03 honesty and truthfulness', regex=False)\
+        .str.replace('rule: 2: moral conduct;paragraph 01 adherence to law',
+                     'rule 2: moral conduct;paragraph 01 adherence to law', regex=False)\
+        .str.replace('rule 6: official info ; paragraph 02 false or inaccurate reports',
+                     'rule 6: official info;paragraph 02 false or inaccurate reports', regex=False)\
+        .str.replace(r'^rule 2: moral conduct ; paragraph 01 adhÃ©rence to law$',
+                     'rule 2: moral conduct ; paragraph 01 adherence to law')
```

Comment:
These lines look like they could be condensed to just a few lines with proper use of regex.